src/posix.mak

```diff
@@ -466,25 +466,20 @@ $G/idgen: $D/idgen.d $(HOST_DMD_PATH)
 	CC=$(HOST_CXX) $(HOST_DMD_RUN) -of$@ $<
 	$G/idgen
 
-#########
-# STRING_IMPORT_FILES
-#
-# Create (or update) the generated VERSION file.
-# The file is only updated if the VERSION file changes, or, only when RELEASE=1
-# is not used, when the full version string changes (i.e. when the git hash or
-# the working tree dirty states changes).
-# The full version string have the form VERSION-devel-HASH(-dirty).
-# The "-dirty" part is only present when the repository had uncommitted changes
-# at the moment it was compiled (only files already tracked by git are taken
-# into account, untracked files don't affect the dirty state).
-VERSION := $(shell cat ../VERSION)
-ifneq (1,$(RELEASE))
-VERSION_GIT := $(shell printf "`$(GIT) rev-parse --short HEAD`"; \
-       test -n "`$(GIT) status --porcelain -uno`" && printf -- -dirty)
-VERSION := $(addsuffix -devel$(if $(VERSION_GIT),-$(VERSION_GIT)),$(VERSION))
+######## VERSION
+
+VERSION := $(shell cat ../VERSION) # default to checked-in VERSION file
+ifneq (1,$(RELEASE)) # unless building a release
```

Comment:
Ok, but going on the other use of RELEASE in the makefile it is 1 when building a release.
https://github.com/MartinNowak/dmd/blob/c28e2d8cfebfcc5b2fb8ee4b35bfb661314bc042/src/posix.mak#L178

So `ifneq (1,$(RELEASE))` would indicate "Do this when not building a release", which doesn't match the comment. This would mean either the code or the comment is wrong.